railties/test/application/test_runner_test.rb

```diff
@@ -249,6 +252,73 @@ def test_generated_scaffold_works_with_rails_test
       assert_match "0 failures, 0 errors, 0 skips", run_test_command('')
     end
 
+    def test_run_multiple_folders
+      create_test_file :models, 'account'
+      create_test_file :controllers, 'accounts_controller'
+
+      run_test_command('test/models test/controllers').tap do |output|
+        assert_match 'AccountTest', output
+        assert_match 'AccountsControllerTest', output
+        assert_match '2 runs, 2 assertions, 0 failures, 0 errors, 0 skips', output
+      end
+    end
+
+    def test_run_multiple_files_and_run_one_file_by_line_only_runs_filter
```

Comment:
@kaspth I'm not sure we need to support that. What got my attention is that these tests specify behavior that is caused by the current implementation not because we designed it like that. I think the behavior is surprising, you can pass arguments that are not at all considered. I think it would be better to either warn the user when that happens or get it working.